pycbc/events/eventmgr.py

```diff
@@ -340,6 +340,10 @@ def finalize_template_events(self):
         self.accumulate.append(self.template_events)
         self.template_events = numpy.array([], dtype=self.event_dtype)
 
+    def consolidate_events(self):
```

Comment:
Ah, I forgot I changed this so self.events wasn't being populated initially anyway. This seems like a reasonable approach.